src/main/resources/view/DarkTheme.css

```diff
@@ -1,18 +1,26 @@
 .root {
-    -COLOR_PRIMARY: #101010;
-    -COLOR_PRIMARY_ONE: #282828;
-    -COLOR_PRIMARY_TWO: #404040;
-    -COLOR_PRIMARY_THREE: #585858;
+    /*
+    Colors inspired by material design principles which can be found at
+    https://material.io/design/color/dark-theme.html.
+    DP refers to the elevation level as stated on the website.
+    */
+
+    -BASE_COLOR: #121212;
+    -DP_0: rgba(255, 255, 255, 0.00);
```

Comment:
I think it would be better if DP can be labelled better. Like DP_COLOR_NO_TRANSPARENCY etc

Reply:
dp refers to elevation as per comments, i think is fine for now. If not it'll be DP_COLOR_LOWEST, DP_COLOR_SLIGHTLY_HIGHER, DP_COLOR_SLIGHTLY_SLIGHTLY_HIGHER, which is not sustainable

Reply:
Got it, then DP_1_ELEVATION would be good

Reply:
alright i'll make a quick change